kuma/static/styles/dashboards.styl

```diff
@@ -53,7 +53,7 @@
 
     /* Select only the outer table cells. */
     > thead th, .dashboard-row > td {
-        @extend $column-strip;
+        @extend $column-2;
```

Comment:
I think we need to deal with the extra table column in a different way. I'll add a commit.